tools/verify-phrases.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
```

Comment:
This script will check to make sure all uses of the RFC2119 keywords are upper-cased. This will help make sure that we don't include them by mistake.

Reply:
@duglin there are a lot of changes from top of file to ### namespace -- pulling those into a separate PR will make it easier for reviewer to focus on the things that could potentially change the meaning of the spec

Reply:
yes but a lot of them are related to removing rfc2119 keywords in places that we didn't mean to use them.

Reply:
True, though just thought the reviews might be easier in two parts that are logically separate, even though they are related.  Just a suggestion.